python/tk_rv/menu_generation.py

```diff
@@ -86,21 +86,27 @@ def create_menu(self):
         # commands_by_menu = {
         #     "Flow Production Tracking":[menu_item, ...],
         # }
+
         for cmd in menu_commands:
             menu_item = cmd.define_menu_item()
-
             command_added = False
             if menu_overrides:
+
                 for menu_override, commands in menu_overrides.items():
                     app_name = cmd.get_app_name()
-                    if app_name in [
-                        c.get("app_instance")
-                        for c in commands
-                        if cmd.name == c.get("name")
-                    ]:
-                        commands_by_menu[menu_override].append(menu_item)
+                    if (menu_override != "SG Review" or "RV_LOAD_SG_REVIEW" in os.environ):
+                        if app_name in [
+                            c.get("app_instance")
+                            for c in commands
+                            if cmd.name == c.get("name")
+                        ]:
+                            commands_by_menu[menu_override].append(menu_item)
+                            command_added = True
+                            break
+                    else:
+
+                        # command_added to true in order to not add the default menu item
```

Comment:
Not a blocker but I'd prefer to limit the size of this change and avoid creating nested conditional blocks.


```suggestion
                    if menu_override == "SG Review" or "RV_LOAD_SG_REVIEW" not in os.environ):
                      # command_added to true in order to not add the default menu item
                      command_added = True
                      continue

                    if app_name in [
                        c.get("app_instance")
                        for c in commands
                        if cmd.name == c.get("name")
                    ]:
                        commands_by_menu[menu_override].append(menu_item)
                        command_added = True
                        break     
```

Reply:
Looks good to me! Just tested that as well.